react/introduction/setting_up_a_react_environment.md

```diff
@@ -0,0 +1,145 @@
+### Introduction
+
+You now know _what_ React is, you might even have an idea of _where_ to use it, it's time to start figuring out _how_. 
+
+This lesson is going to cover starting a new React project on your own machine, as well as some useful tools to help you along the way. We'll also explain some of the problems that may arise (and how to avoid them).
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+* How React projects can be created locally.
+* Describing the difficulties when starting a React project without the appropriate tools.
+* Listing some of the popular tools and methods for scaffolding a new React project.
+* Explaining what Create React App is.
+* Setting up your workflow to develop efficiently within your new project.
+
+### Many Paths
+
+There are multiple ways to start using React in your projects, from attaching a set of `<script>` tags which serve React from a [CDN](https://en.wikipedia.org/wiki/Content_delivery_network), to robust [toolchains](https://en.wikipedia.org/wiki/Toolchain) that are highly configurable and allow for increased scalability and optimization.
+
+Some examples of these toolchains:
+
+* [Create React App](https://create-react-app.dev/)
+* [Vite's React Config](https://vitejs.dev/)
+* [Gatsby](https://www.gatsbyjs.com/)
+* [NextJS](https://nextjs.org/)
+
+Why do we need these toolchains? Can't we just make our own as we see fit?
+
+Yes, but it's _hard_. React is a complex beast and there are many moving parts. Before you can start writing any sort of code that provides functionality, you would need to configure _at least_ the following:
+
+* Package Management (NPM, Yarn)
+* Module bundling (Webpack, Parcel)
+* Compilation (Babel)
+* React itself
+
+All of this, and sometimes _much more_ is required to get a React project and development environment up and running.
+
+### Simplifying The Process
+
+Now that you understand what is involved with starting a React project from scratch, you can breathe a sigh of relief as you learn that we can get started with a _single terminal command_.
+
+This terminal command is a feature of the beginner-friendly [Create React App](https://create-react-app.dev/), a tool developed by the folks over at Meta which we will be using from here on out. It requires minimal configuration and provides extremely useful tools straight out of the box, allowing us to get straight to the learning. Let's get started!
```

Comment:
Adding an abbreviation for Create React App here so that we can just say CRA instead of "Create React App":

```suggestion
This terminal command is a feature of the beginner-friendly [Create React App](https://create-react-app.dev/), or CRA for short, a tool developed by the folks over at Meta which we will be using from here on out. It requires minimal configuration and provides extremely useful tools straight out of the box, allowing us to get straight to the learning. Let's get started!
```

Reply:
Love it, done.